src/generic/hooks/index.tsx

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Once [this](https://github.com/openedx/frontend-build/pull/609) merges we should be able to bump frontend-build and get rid of lines like this.